src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -73,6 +74,10 @@ private async Task<IAccount> GetAccountFromParamsOrLoginHintAsync(AcquireTokenSi
 
         internal async override Task PreRunAsync()
         {
+#if ANDROID
```

Comment:
We need a better abstraction here . Can we have a param on the Broker class or on the PlatformProxy class that knows when to invoke the ios / android broker, smth like `if broker_supports_silent_auth then ...`